src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * AddCommand class that encompasses the todo, deadline, event commands
+ */
+public class AddCommand extends Command {
+    protected String fullCommand;
+    protected String[] s;
```

Comment:
Perhaps a better variable name can be used here, i.e. parsedCommand/splitCommand etc.